### DISCUSSION TITLE
[365450] hacknet本体

### DISCUSSION THREAD

**Discussion by z0d0dOr** [pjy612/SteamManifestCache#202] [Share]
`appid:365450`
keys:
```
365451;2d05152a1b87ac918aab52f39201a9d630db7062613a265cb84f4d4f3540f48e
```
code:`74430CCDC69A4EB3`
email:zddrgithub@gmail.com

**Comment by z0d0dOr** [comment]
请问不可以参加内测了嘛，为啥done了但是没有收到邮件呢

**Reply by pjy612** [reply]
![image](https://github.com/pjy612/SteamManifestCache/assets/4072526/ef6dcc52-ba1e-49d3-9f13-b83ffc117d85)

**Reply by z0d0dOr** [reply]
跑到垃圾邮件了哈哈
![image](https://github.com/pjy612/SteamManifestCache/assets/128966342/68f293ff-c735-446b-adab-fee88f517d63)

**Comment by z0d0dOr** [comment]
[hacknet.zip](https://github.com/pjy612/SteamManifestCache/files/13797488/hacknet.zip)
补个清单
